### PR TITLE
fix (webapi): add searchbox gui element to hasOtherSomething property

### LINKED ISSUE #827: SALSAH V1 GUI: has other something
It seems that that the GUI cannot figure out the value type of `something:hasOtherSomething`. Is this a bug?

something

### PR DESCRIPTION
**Pull request by musicEnfanthen**
This PR fixes the missing searchbox gui element for `hasOtherSomething` property in the something ontology.

fixes #827

### CODE REVIEW

**Comment by musicEnfanthen** [comment]
Fixes #827

**Comment by musicEnfanthen** [comment]
Ready to view.

File: webapi/_test_data/ontologies/something-onto.ttl [hunk 1]
```diff
@@ -65,4 +67,6 @@
 
 	knora-base:subjectClassConstraint :Something ;
 
-	knora-base:objectClassConstraint knora-base:LinkValue .
+	knora-base:objectClassConstraint knora-base:LinkValue ;
+
+	salsah-gui:guiElement salsah-gui:Searchbox .
```
Comment (tobiasschweizer) [inline webapi/_test_data/ontologies/something-onto.ttl]: Could you add a newline at the end of the file?

**Comment by tobiasschweizer** [comment]
I ran the browser tests for salsah1 and they passed :-) 

see #829

File: webapi/_test_data/ontologies/something-onto.ttl (hunk 1 above)
Comment (musicEnfanthen) [reply webapi/_test_data/ontologies/something-onto.ttl]: Sorry I missed that. Added the new line right now.

Comment (tobiasschweizer) [reply webapi/_test_data/ontologies/something-onto.ttl]: Thanks! It shouldn't matter, the triplestore did not complain ;-) It is rather a issue of style.

**Pull request merged**